src/views/pages/About/index.jsx

```diff
@@ -0,0 +1,26 @@
+import { useScreenSize } from "../../../hooks/useScreenSize";
+import { useNavigate } from "react-router-dom";
+import { HomeWrapper, StyledLogo, BackIconz } from "./styled.components";
+import AboutContent from "../../fragments/AboutContent";
+import BackIcon from "../../assets/icons/back-icon2.png";
+
+const About = () => {
+  const navigate = useNavigate(); 
+  const onHomeClick = () => {
+    navigate("/");
+  };
+  const { isSmallScreen } = useScreenSize();
+  return (
+    <HomeWrapper>
+      {isSmallScreen && (
+        <>
+          <BackIconz src= {BackIcon} onClick = {onHomeClick} />
```

Comment:
instead of adding the source here, if we are handling svg vectors we can use them as react components, then importing the styled component only